modules/odc-using-the-developer-catalog-to-add-services-or-components.adoc

```diff
@@ -9,8 +9,8 @@ You use the Developer Catalog to deploy applications and services based on Opera
 
 .Procedure
 
-. In the *Developer* perspective, navigate to the *+Add* -> *From Catalog* to view all the available services in the *Developer Catalog*.
-. Under *All Items*, select the kind of service or the component you need to add to your project. For this example, select *Databases* to list all the database services and then click *MariaDB* to see the details for the service.
+. In the *Developer* perspective, navigate to the *+Add* view and from the *Developer Catalog* tile, click on *All Services* to view all the available services in the *Developer Catalog*.
+. Under *All Services*, select the kind of service or the component you need to add to your project. For this example, select *Databases* to list all the database services and then click *MariaDB* to see the details for the service.
 +
 .Developer Catalog
```

Comment:
Did All Items change to *All Services*? If yes please remove the screenshot below, it shows All Items. Use screenshots only to explain very complicated user flows.

Reply:
Done!